checkmk_extensions/agent/robotmk_collector.ps1

```diff
@@ -27,6 +24,24 @@ function Main {
 }
 
 
+function DetermineConfigPath {
+    [OutputType([string])]
+    Param (
+        [parameter(Mandatory=$false, Position=0)]
+        [String[]]$CommandLineArgs
+    )
+
+    if ($CommandLineArgs -ne "" -and $CommandLineArgs.Count -gt 0) {
```

Comment:
I needed to change this to `$CommandLineArgs -ne ""` because now, if there are no arguments provided, it considers it as an empty string.